src/app/modules/auth/authRoutes.ts

```diff
@@ -0,0 +1,25 @@
+import {LoginDTO} from './dtos/LoginDTO'
+import {RegisterDTO} from './dtos/RegisterDTO'
+import {AuthController} from './AuthController'
+import {AuthService} from '../../services/auth/AuthService'
+import {bindDependencies} from '../../dependencies/utils'
+
+export default [
+  {
+    method: 'POST',
+    path: '/auth/register',
+    schema: RegisterDTO.schema,
+    handler: AuthController.register
+  },
+  {
+    method: 'POST',
+    path: '/auth/token',
+    schema: LoginDTO.schema,
+    handler: bindDependencies(AuthController.login, AuthService)
+  },
+  {
+    method: 'GET',
+    path: '/auth/token',
+    handler: bindDependencies(AuthController.refresh, AuthService)
```

Comment:
The function-level injection seems slightly odd to me, but this is somewhat new territory. I feel like if we mess around with the controller class we could get this slightly cleaner.

Reply:
The injectify homie doesn't seem to have a good story for injecting into static methods, because it's all about instantiating objects and injecting in the constructor. But like I said above, static methods are just functions, so this felt simpler.

Reply:
Yeah I mean it's simple at least.